properties.go

```diff
@@ -94,6 +94,15 @@ const (
 	// multicast on startup.
 	EnvVarMulticastEnabled = "SMUDGE_MULTICAST_ENABLED"
 
+	// EnvVarMulticastAnnounceInterval is the name of the environment variable
+	// that describes whether Smudge will attempt to re-announce its presence
+	// via multicast every X seconds.
+	EnvVarMulticastAnnounceInterval = "SMUDGE_MULTICAST_ANNOUNCE_INTERVAL"
+
+	// DefaultMulticastAnnounceIntervalSeconds is the default value for whether Smudge
+	// will re-announce its presence via multicast
+	DefaultMulticastAnnounceIntervalSeconds = 0
+
 	// DefaultMulticastEnabled is the default value for whether Smudge will
 	// attempt to announce its presence via multicast on startup.
 	DefaultMulticastEnabled string = "true"
```

Comment:
Can `DefaultMulticastEnabled` be moved up so that it follows its corresponding `EnvVar`?